app/models/legacy_ect_profile.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
-class EarlyCareerTeacherProfile < ApplicationRecord
+class LegacyECTeacherProfile < ApplicationRecord
+  self.table_name = "early_career_teacher_profiles"
```

Comment:
When do we axe them?

Reply:
Soon - I just keep them to ensure there are no data missing after the switch.